redis-hbo-provider/src/test/java/com/facebook/presto/statistic/TestHistoricalStatisticsSerde.java

```diff
@@ -30,7 +30,7 @@
 public class TestHistoricalStatisticsSerde
 {
     @Test
-    public void TestSimpleHistoricalStatisticsEncoderDecoder()
+    public void testSimpleHistoricalStatisticsEncoderDecoder()
```

Comment:
This test is missing an assertion.  The last line should be `assertEquals(x, y)` and not `x.equals(y)`